wiki/Manual.md

```diff
@@ -5061,7 +5060,7 @@ fetchLedgerEntry (id, code = undefined, params = {})
 Parameters
 
 - **id** (String) *required* Ledger entry id
-- **code** (String) Unified CCXT currency code, required (e.g. `"USDT"`)
+- **code** (String) *required* Unified CCXT currency code (e.g. `"USDT"`)
```

Comment:
code might or might not be required depending on the exchange (for instance bitmex/binance/coinbase/etc) do not require a code